src/porepy/numerics/linalg/matrix_operations.py

```diff
@@ -834,3 +834,22 @@ def rldecode(A: np.ndarray, n: np.ndarray) -> np.ndarray:
     j[i[1:-1:]] = 1
     B = A[np.cumsum(j)]
     return B
+
+
+def sparse_kronecker_product(matrix: sps.spmatrix, nd: int) -> sps.spmatrix:
```

Comment:
Do we need a test for this? I'm inclined to say yes, but testing 2-3 cases should be sufficient.

Reply:
What would you test?